docs/development.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Development"
+date: 2022-01-28T00:00:00+00:00
+weight: 50
+geekdocRepo: https://github.com/owncloud/web
+geekdocEditPath: edit/master/docs
+geekdocFilePath: development.md
+---
+
+{{< toc >}}
+
+This is a collection of tips and conventions to follow when working on [the ownCloud web frontend](https://github.com/owncloud/web). 
+It is a living document, please open a PR if you find something missing.
+
+## Contributing to ownCloud web
+
+Everyone is invited to contribute. Simply fork [the codebase](), check [the issues](https://github.com/owncloud/web/issues?q=is%3Aopen+is%3Aissue+label%3ATopic%3Agood-first-issue) for a suitable one and open a pull request!
+
+### Linting and tests
+
+To make sure your pull request can be efficiently reviewed and won't need a lot of changes down the road, please run the linter and the unit tests via `yarn lint --fix` and `yarn test:unit` locally. [Our CI](https://drone.owncloud.com/owncloud/web) will run on pull requests and report back any problems after that.
```

Comment:
event if we're not sure how to proceed with the integration tests... but i think right now we still have to mention those. Thoughts!? Also can we link the testing page here?

Reply:
Adding integration tests & the link to the testing page, great suggestion